src/applications/login/containers/SignInApp.jsx

```diff
@@ -61,6 +61,7 @@ class SignInPage extends React.Component {
     const { query } = this.props.location;
     const loggedOut = query.auth === 'logged_out';
     const application = query.application;
+    const redirect = query.to;
```

Comment:
Does this allow relative paths only? Like, if someone types `?to=https://malicious-site.com`, that wouldn't redirect?

Reply:
Yes, but that's enforced in the corresponding vets-api PR (link in description) where we assemble the full redirect path from a base URL derived from the `application` parameter plus the `to` parameter.